exercises/atbash-cipher/cipher.t

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env perl6
+
+use Test;
+use JSON::Tiny;
+
+use lib ( my $dir = IO::Path.new($?FILE).parent ).path;
+
+my $module_name = %*ENV<EXERCISM>.so ?? 'Example' !! 'Allergies';
+my @potential_module = <p6 pm6 pm>.map:  $module_name ~ '.' ~ *; 
+
+my $module = first { $dir.child($_).e }, |@potential_module
+    or die "No file '$module_name.p6' found\n";
+
+require $module <&encode &decode>;
+
+plan 2;
+
+my %cases = from-json $dir.child('cases.json').slurp;
+
+subtest 'encode' => {
+    my @cases = |%cases.<encode>.<cases>;
+
+    plan +@cases;
+
+    is encode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+subtest 'decode' => {
+    my @cases = |%cases.<decode>.<cases>;
+
+    plan +@cases;
+
+    is decode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+done-testing;
```

Comment:
`done-testing` is usually for when you can't/don't have a `plan`, I don't think it's a good idea to keep them both together. The testing docs recommend that it be removed so I don't think we should keep it in without a good reason for doing so.